### PR TITLE
[E2E]add worker group manage e2e case

### PR DESCRIPTION
**Pull request by janeHe13**
## Purpose of the pull request
add e2e security case #7281 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by caishunfeng — DISMISSED** [review]
+1

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/WorkerGroupE2ETest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    private static RemoteWebDriver browser;
+
+    @BeforeAll
+    public static void setup() {
+        new LoginPage(browser)
+            .login("admin", "dolphinscheduler123")
+                .goToNav(SecurityPage.class)
+                .goToTab(TenantPage.class)
+                .create(tenant)
```
Comment (zhongjiajie) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/WorkerGroupE2ETest.java]: NIT, indentation
```suggestion
            .goToNav(SecurityPage.class)
            .goToTab(TenantPage.class)
            .create(tenant)
```

**Review by zhongjiajie — COMMENTED** [review]
Add NIT comment

**Review by songjianet — DISMISSED** [review]
LGTM

**Review by songjianet — APPROVED** [review]

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**